Adamant/Modules/AlertsFactory/FreeTokensAlertManager.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  FreeTokensAlertManager.swift
+//  Adamant
+//
+//  Created by Владимир Клевцов on 23.1.25..
+//  Copyright © 2025 Adamant. All rights reserved.
+//
+import UIKit
+import SafariServices
+import CommonKit
+
+enum AlertPresenter {
+    @MainActor
+    static func freeTokenAlertIfNeed(type: FreeTokensAlertType) {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        let accountService = appDelegate.container.resolve(AccountService.self)!
+        let currentBalance = accountService.account?.balance
+        if currentBalance ?? 0.0 < AdamantApiService.KvsFee {
```

Comment:
let's rewrite it so we should not think about operations priority:
`if let currentBalance, currentBalance < AdamantApiService.KvsFee`